.chloggen/fix-public-api-otlpreceiver.yaml

```
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

I don't think it's a breaking change. We expose additional API, nothing else is affected.
```suggestion
change_type: enhancement
```